.buildkite/release-automation/forge_x86_64.Dockerfile

```diff
@@ -13,6 +13,11 @@ apt-get update
 apt-get upgrade -y
 apt-get install -y curl zip clang-12 git
 
+# Needs to be synchronized to the host group id as we map /var/run/docker.sock
+# into the container.
+addgroup --gid 1001 docker0  # Used on old buildkite AMIs.
```

Comment:
you don't need the lines related to `docker0` now. we have completed the migration 3 months ago.

could you remove the `docker0` related lines?

Reply:
done